packages/kubernetes-1.17/kubelet-env

```diff
@@ -2,3 +2,4 @@ NODE_IP={{settings.kubernetes.node-ip}}
 NODE_LABELS={{join_map "=" "," "no-fail-if-missing" settings.kubernetes.node-labels}}
 NODE_TAINTS={{join_map "=" "," "no-fail-if-missing" settings.kubernetes.node-taints}}
 POD_INFRA_CONTAINER_IMAGE={{settings.kubernetes.pod-infra-container-image}}
+CLOUD_PROVIDER={{settings.kubernetes.cloud-provider}}
```

Comment:
Same question as above. Curious whether this needs to be an environment variable whether its value should be written to the conf.

Reply:
Everything but `POD_INFRA_CONTAINER_IMAGE` can be in the conf template now. The environment variable approach was all we had before ef41a6b1, but isn't as flexible.

We don't have to move the other settings here, but shouldn't add more.

Reply:
Updated.